extensions/wikia/PortableInfoboxBuilder/services/Nodes/NodeGroup.class.php

```diff
@@ -16,7 +16,26 @@ class NodeGroup extends Node {
 	protected $requiredChildNodes = [ 'header' ];
 
 
+	/**
+	 * @var array of string
+	 */
+	protected $allowedAttributes = [ 'collapse' ];
```

Comment:
We need to have check for supported (allowed) values of this attribute too (also in the tests)